vulcanai2/models/basenetwork.py

```diff
@@ -498,6 +499,20 @@ def _validate(self, val_loader):
         validation_accuracy = val_accuracy_accumulator * len(data) / len(val_loader.dataset)
 
         return validation_loss, validation_accuracy
+    
```

Comment:
I think when you apply cuda to a model, it automatically applies it to incoming models

Reply:
NopeI get this `RuntimeError: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same`
Thats because our model workflow involves lists of model?. I think so.